### ISSUE TITLE
关于dpvs无法运行的相关问题

### ISSUE THREAD

**Issue by SpiritComan** [iqiyi/dpvs#77]
尊敬的爱奇艺开发者们:
       你们好，首先非常感谢你们提供、开源的这个程序。我跟我的小组依照你们的程序只是，在虚拟机以及实体机上都进行了安装测试，但都会报出“EAL: Error - exiting with code: 1 Cause: Cannot init mbuf pool on socket 1”错误，导致程序无法顺利进行。我想请问一下是否dpvs对环境、底层硬件依赖性太强？是否虚拟机下无法正确运行？是否实体机的硬件环境与你们不一致的时候无法运行？希望得到你们的回答，万分感激！

**Comment by beacer** [comment]
Can you run DPDK-16.07 example like `l2fwd` on you VM or physical machine ?

**Comment by SpiritComan** [comment]
Yes,i run l2fwd like this:./build/app/l2fwd -l 0-3 -n 4 -- -q 8 -p ffff,the program showed the msg like the image show。
![image](https://user-images.githubusercontent.com/26474049/34348097-351c1348-ea43-11e7-8958-7337196b895d.png)

**Comment by beacer** [comment]
你可以试试调整下，hugepage的大小。或者帮忙debug一下，因为我们这里没有出现这个问题，光看log不好说是为啥。
```
$ echo 8192 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages
$ echo 8192 > /sys/devices/system/node/node1/hugepages/hugepages-2048kB/nr_hugepages
```
